File: programacion/php/paneladmin/clases/condbusu.php
<?php


namespace clasespanel;


class condbusu
{
    private $dbconn;

    public function __construct($dbconn)
    {
        $this->dbconn = $dbconn;
    }

// Consulta de los usuarios creados en la base de datos
    function listausuarios(){

        $listausu = $this->dbconn->query("select usuario from desmicusuarios");
        $usuarios = array();
        while ($row = $listausu->fetch_array()[0]) {
            array_push($usuarios, $row);
        }

        return $usuarios;
    }

// Consulta de los perfiles creados en la base de datos
    function listaperfiles(){

        $listaper = $this->dbconn->query("select perfil from desmicperfiles");
        $perfiles = array();
        while ($row = $listaper->fetch_array()[0]) {
            array_push($perfiles, $row);
        }

        return $perfiles;
    }

// Consulta de las acciones creadas en la base de datos
    function listaacciones(){

        $listaacc = $this->dbconn->query("select accion from desmicacciones");
        $acciones = array();
        while ($row = $listaacc->fetch_array()[0]) {
            array_push($acciones, $row);
        }

        return $acciones;
    }
}

File: programacion/python/paneladmin/clases/condbusu.py
import mysql.connector


class condbusu:

    def __init__(self, dbconn):
        self.dbconn = dbconn

# Consulta de los usuarios creados en la base de datos   
    def listausuarios(self):
        dbconncursor = self.dbconn.cursor()
        dbconncursor.execute("select usuario from desmicusuarios")

        listausu = dbconncursor.fetchall()
        usuarios = []
        
        for i in range(len(listausu)):
            usuarios.append(listausu[i][0])
        
        return usuarios

# Consulta de los perfiles creados en la base de datos    
    def listaperfiles(self):
        dbconncursor = self.dbconn.cursor()
        dbconncursor.execute("select perfil from desmicperfiles")

        listaper = dbconncursor.fetchall()
        perfiles = []
        
        for i in range(len(listaper)):
            perfiles.append(listaper[i][0])
        
        return perfiles

# Consulta de las acciones creadas en la base de datos    
    def listaacciones(self):
        dbconncursor = self.dbconn.cursor()
        dbconncursor.execute("select accion from desmicacciones")

        listaacc = dbconncursor.fetchall()
        acciones = []
        
        for i in range(len(listaacc)):
            acciones.append(listaacc[i][0])
        
        return acciones